cpp/src/aztec/honk/composer/composer_helper/standard_honk_composer_helper.cpp

```diff
@@ -28,7 +28,7 @@ std::shared_ptr<bonk::proving_key> StandardHonkComposerHelper<CircuitConstructor
     const CircuitConstructor& constructor, const size_t minimum_circuit_size, const size_t num_randomized_gates)
 {
     // Initialize circuit_proving_key
-    // TODO: replace composer types.
+    // TODO(Kesha): replace composer types.
```

Comment:
@Rumata888 is this note saying that `ComposerType` is not the right thing anymore, and we need a class like `Arithmetization` or `ConstructorType` or something? If so, we could link to #229, which I just created.

Reply:
This now references 229